src/Entities/Categories.php

```diff
@@ -1,6 +1,31 @@
 <?php
 
-class Categories
+namespace Fsbe\Entities;
+
+use JsonSerializable;
+
+class Categories implements JsonSerializable
 {
+    private array $categories;
+
+    /**
+     * @return array
+     */
+    public function getCategories(): array
+    {
+        return $this->categories;
+    }
+
+    /**
+     * @param array $categories
+     */
+    public function setCategories(array $categories): void
```

Comment:
this means its a one and done addition, might be best to amend this so that categories can be continually added if needs be rather than steamrolling the property each time